example/src/views/login/screen2.js

```diff
@@ -1,45 +1,416 @@
-import React, { Component } from 'react';
```

Comment:
Just fyi, the above images are too big. 512 x 512. In the future compress them down to a smaller 128 x 128 & also provide a @2x & @3x for different phone sizes.

This might be useful to read: https://facebook.github.io/react-native/docs/images.html#static-image-resources